picard/ui/metadatabox.py

```diff
@@ -278,6 +282,10 @@ def __init__(self, parent=None):
         self.ignore_updates = IgnoreUpdatesContext(on_exit=self.update)
         self.tagger.clipboard().dataChanged.connect(self._update_clipboard)
 
+    def _on_setting_changed(self, key, old_value, new_value):
+        if key in ('rename_files', 'move_files'):
```

Comment:
It may not make much of a difference but this might be more efficient if the keys to check were in a set rather than a tuple.

Reply:
Done.